packages/fxa-settings/src/pages/ResetPasswordRedesign/ResetPassword/container.tsx

```diff
@@ -0,0 +1,59 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { RouteComponentProps, useLocation, useNavigate } from '@reach/router';
+import React, { useState } from 'react';
+import { getLocalizedErrorMessage } from '../../../lib/auth-errors/auth-errors';
+import { useAuthClient, useFtlMsgResolver } from '../../../models';
+
+import { ResetPasswordContainerProps } from './interfaces';
+import { queryParamsToMetricsContext } from '../../../lib/metrics';
+import ResetPassword from '.';
+
+const ResetPasswordContainer = ({
+  flowQueryParams = {},
+  serviceName,
+}: ResetPasswordContainerProps & RouteComponentProps) => {
+  const authClient = useAuthClient();
+  const ftlMsgResolver = useFtlMsgResolver();
+  const location = useLocation();
+  const navigate = useNavigate();
+
+  const searchParams = location.search;
+
+  const [errorMessage, setErrorMessage] = useState<string>('');
+
+  let localizedErrorMessage = '';
+
+  const requestResetPasswordCode = async (email: string) => {
+    const metricsContext = queryParamsToMetricsContext(
+      flowQueryParams as unknown as Record<string, string>
+    );
+    const options = {
+      metricsContext,
+    };
+    try {
+      await authClient.passwordForgotSendOtp(email, options);
+      navigate(`/confirm_reset_password${searchParams}`, {
+        state: { email, metricsContext },
+      });
+    } catch (err) {
+      localizedErrorMessage = getLocalizedErrorMessage(ftlMsgResolver, err);
```

Comment:
Do we expect this to happen? Perhaps record Sentry event if this is unexpected so we can track it?

Reply:
This could happen due to throttling on `passwordForgotSendOtp`, for example, so we would expect to show a user-facing message in that instance. If there are issues on the backend, those would already be recorded in Sentry, correct?

Reply:
Any unexpected error would get recorded by `auth-server`; however, expected error states like throttling errors (from customs), or even a payload validation error that you might not have expected to occur would not. It sounds like we have good handle on the potential failures here and it's not necessary.